doc/tutorial/using_gpu.txt

```diff
@@ -470,7 +470,7 @@ the GPU object directly.  The following code is modifed to do just that.
 
   rng = numpy.random.RandomState(22)
   x = shared(numpy.asarray(rng.rand(vlen), config.floatX))
-  f = function([], sandbox.gpuarray.basic_ops.gpu_from_host(tensor.exp(x)))
+  f = function([], sandbox.gpuarray.basic_ops.GpuFromHost(None)(tensor.exp(x)))
```

Comment:
For now, how about we add an instance "gpu_from_host = GpuFromHost(None)" to gpuarray.basic_ops?